### ISSUE TITLE
dbeaver crashing on start - archlinux 

### ISSUE THREAD

**Issue by sr523** [dbeaver/dbeaver#10837]
On start of dbeaver, this error is being thrown:

Splash screen opens and then application crashes.  The log output below is truncated for brevity.  It repeats multiple times for each eclipse unit. 

pacman - https://archlinux.org/packages/community/x86_64/dbeaver/

```
!SESSION 2020-12-30 10:33:01.905 -----------------------------------------------
eclipse.buildId=unknown
java.version=1.8.0_265
java.vendor=Oracle Corporation
BootLoader constants: OS=linux, ARCH=x86_64, WS=gtk, NL=en_US
Command-line arguments:  -os linux -ws gtk -arch x86_64

!ENTRY org.eclipse.compare 4 0 2020-12-30 10:33:03.373
!MESSAGE FrameworkEvent ERROR
!STACK 0
org.osgi.framework.BundleException: Could not resolve module: org.eclipse.compare [27]
  Unresolved requirement: Require-Bundle: org.eclipse.ui.editors; bundle-version="[3.5.0,4.0.0)"
    -> Bundle-SymbolicName: org.eclipse.ui.editors; bundle-version="3.13.400.v20200923-1238"; singleton:="true"
       org.eclipse.ui.editors [146]
         Unresolved requirement: Require-Bundle: org.eclipse.ui.ide; bundle-version="[3.5.0,4.0.0)"
           -> Bundle-SymbolicName: org.eclipse.ui.ide; bundle-version="3.18.0.v20201119-1132"; singleton:="true"
              org.eclipse.ui.ide [148]
                Unresolved requirement: Require-Bundle: org.eclipse.ui.workbench; bundle-version="[3.115.100,4.0.0)"
                  -> Bundle-SymbolicName: org.eclipse.ui.workbench; bundle-version="3.122.0.v20201122-1345"; singleton:="true"
                     org.eclipse.ui.workbench [158]
                       Unresolved requirement: Import-Package: org.eclipse.e4.core.commands.internal
                         -> Export-Package: org.eclipse.e4.core.commands.internal; bundle-symbolic-name="org.eclipse.e4.core.commands"; bundle-version="0.13.0.v20201119-1132"; version="0.0.0"; x-friends:="org.eclipse.e4.ui.bindings,org.eclipse.ui.workbench"
                            org.eclipse.e4.core.commands [45]
                              Unresolved requirement: Require-Capability: osgi.ee; filter:="(&(osgi.ee=JavaSE)(version=11))"
                       Unresolved requirement: Import-Package: org.eclipse.e4.core.commands
                         -> Export-Package: org.eclipse.e4.core.commands; bundle-symbolic-name="org.eclipse.e4.core.commands"; bundle-version="0.13.0.v20201119-1132"; version="0.0.0"; x-friends:="org.eclipse.e4.ui.bindings,  org.eclipse.e4.ui.workbench,  org.eclipse.e4.ui.workbench.renderers.swt,  org.eclipse.ui.workbench,  org.eclipse.e4.ui.workbench.swt,  org.eclipse.e4.ui.progress,  org.eclipse.e4.core.commands.tests"
                Unresolved requirement: Require-Bundle: org.eclipse.ui; bundle-version="[3.106.0,4.0.0)"
                  -> Bundle-SymbolicName: org.eclipse.ui; bundle-version="3.118.100.v20201123-0719"; singleton:="true"
                     org.eclipse.ui [143]
                       Unresolved requirement: Require-Bundle: org.eclipse.ui.workbench; bundle-version="[3.120.0,4.0.0)"; visibility:="reexport"
                         -> Bundle-SymbolicName: org.eclipse.ui.workbench; bundle-version="3.122.0.v20201122-1345"; singleton:="true"
  Unresolved requirement: Require-Bundle: org.eclipse.ui.workbench.texteditor; bundle-version="[3.5.0,4.0.0)"
    -> Bundle-SymbolicName: org.eclipse.ui.workbench.texteditor; bundle-version="3.15.100.v20201116-1727"; singleton:="true"
       org.eclipse.ui.workbench.texteditor [159]
         Unresolved requirement: Require-Bundle: org.eclipse.ui; bundle-version="[3.5.0,4.0.0)"
           -> Bundle-SymbolicName: org.eclipse.ui; bundle-version="3.118.100.v20201123-0719"; singleton:="true"
  Unresolved requirement: Require-Bundle: org.eclipse.ui.ide; bundle-version="[3.3.0,4.0.0)"
    -> Bundle-SymbolicName: org.eclipse.ui.ide; bundle-version="3.18.0.v20201119-1132"; singleton:="true"
  Unresolved requirement: Require-Bundle: org.eclipse.ui.views; bundle-version="[3.2.0,4.0.0)"
    -> Bundle-SymbolicName: org.eclipse.ui.views; bundle-version="3.10.500.v20201021-1339"; singleton:="true"
       org.eclipse.ui.views [155]
         Unresolved requirement: Require-Bundle: org.eclipse.ui; bundle-version="[3.5.0,4.0.0)"
           -> Bundle-SymbolicName: org.eclipse.ui; bundle-version="3.118.100.v20201123-0719"; singleton:="true"
  Unresolved requirement: Require-Bundle: org.eclipse.ui.forms; bundle-version="[3.2.0,4.0.0)"
    -> Bundle-SymbolicName: org.eclipse.ui.forms; bundle-version="3.11.0.v20201025-0656"; singleton:="true"
       org.eclipse.ui.forms [147]
         Unresolved requirement: Require-Capability: osgi.ee; filter:="(&(osgi.ee=JavaSE)(version=11))"
         Unresolved requirement: Require-Bundle: org.eclipse.e4.ui.css.core; bundle-version="0.10.100"
           -> Bundle-SymbolicName: org.eclipse.e4.ui.css.core; bundle-version="0.13.0.v20201015-0653"; singleton:="true"
              org.eclipse.e4.ui.css.core [54]
                Unresolved requirement: Require-Capability: osgi.ee; filter:="(&(osgi.ee=JavaSE)(version=11))"
         Unresolved requirement: Require-Bundle: org.eclipse.ui; bundle-version="[3.105.0,4.0.0)"; resolution:="optional"
           -> Bundle-SymbolicName: org.eclipse.ui; bundle-version="3.118.100.v20201123-0719"; singleton:="true"
         Unresolved requirement: Require-Bundle: org.eclipse.e4.ui.css.swt; bundle-version="0.11.100"
           -> Bundle-SymbolicName: org.eclipse.e4.ui.css.swt; bundle-version="0.14.0.v20201103-0952"; singleton:="true"
              org.eclipse.e4.ui.css.swt [55]
                Unresolved requirement: Require-Bundle: org.eclipse.e4.ui.css.core; bundle-version="0.12.200"
                  -> Bundle-SymbolicName: org.eclipse.e4.ui.css.core; bundle-version="0.13.0.v20201015-0653"; singleton:="true"
  Unresolved requirement: Require-Bundle: org.eclipse.ui; bundle-version="[3.5.0,4.0.0)"
    -> Bundle-SymbolicName: org.eclipse.ui; bundle-version="3.118.100.v20201123-0719"; singleton:="true"
  Unresolved requirement: Require-Bundle: org.eclipse.ui.navigator; bundle-version="[3.2.0,4.0.0)"; resolution:="optional"
    -> Bundle-SymbolicName: org.eclipse.ui.navigator; bundle-version="3.9.500.v20201021-1339"; singleton:="true"
       org.eclipse.ui.navigator [151]
         Unresolved requirement: Require-Bundle: org.eclipse.ui; bundle-version="[3.6.0,4.0.0)"
           -> Bundle-SymbolicName: org.eclipse.ui; bundle-version="3.118.100.v20201123-0719"; singleton:="true"

	at org.eclipse.osgi.container.Module.start(Module.java:463)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel$2.run(ModuleContainer.java:1845)
	at org.eclipse.osgi.internal.framework.EquinoxContainerAdaptor$1$1.execute(EquinoxContainerAdaptor.java:136)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.incStartLevel(ModuleContainer.java:1838)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.incStartLevel(ModuleContainer.java:1779)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.doContainerStartLevel(ModuleContainer.java:1743)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.dispatchEvent(ModuleContainer.java:1665)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.dispatchEvent(ModuleContainer.java:1)
	at org.eclipse.osgi.framework.eventmgr.EventManager.dispatchEvent(EventManager.java:234)
	at org.eclipse.osgi.framework.eventmgr.EventManager$EventThread.run(EventManager.java:345)
```

**Comment by sr523** [comment]
Fixed by: 
Installed "jre11-openjdk".
followed by "archlinux-java set java-11-openjdk".
Now is running

**Comment by p91paul** [comment]
might be worth opening a bug report to the archlinux package which should depend on jdk >= 11 instead of >= 8 like it is now!

**Comment by pokoli** [comment]
This has been [reported to archlinux bug tracker](https://bugs.archlinux.org/task/69133).

**Comment by muflone** [comment]
> This has been [reported to archlinux bug tracker](https://bugs.archlinux.org/task/69133).

for the records, it was fixed in Arch Linux dbeaver 7.3.1-2
